finance-portal/templates/ingress.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  annotations:
+    kubernetes.io/ingress.class: nginx
+    nginx.ingress.kubernetes.io/rewrite-target: /$1
+  name: {{ .Release.Name }}-ingress-service-portal-backend
+  namespace: {{ .Release.Namespace }}
+spec:
+  rules:
+  - http:
+      paths:
+      - path: /(.*)
+        backend:
+          serviceName: {{ .Release.Name }}-finance-portal
```

Comment:
Why only prefix is being derived from the release?

Please also derive the service name from the helm chart itself.